### PR TITLE
test-images-distributor: no distribution to api.ci

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @stevekuznetsov @alvaroaleman

### CODE REVIEW

File: cmd/dptp-controller-manager/main.go [hunk 1]
```diff
@@ -311,7 +311,9 @@ func main() {
 		if cluster == opts.registryClusterName {
 			registryMgr = mgr
 		} else {
-			allClustersExceptRegistryCluster[cluster] = mgr
+			if cluster != "api.ci" {
+				allClustersExceptRegistryCluster[cluster] = mgr
```
Comment (alvaroaleman) [inline cmd/dptp-controller-manager/main.go:315]: No, do it inside the controller specifically, this map might be used elsewhere

Comment (hongkailiu) [reply cmd/dptp-controller-manager/main.go:315]: Checked. only this controller.
hesitating as well.
Modifying.

**Review by alvaroaleman — APPROVED** [review]

File: pkg/controller/test-images-distributor/test_images_distributor.go [hunk 2]
```diff
@@ -70,6 +70,10 @@ func AddToManager(mgr manager.Manager,
 
 	buildClusters := sets.String{}
 	for buildClusterName, buildClusterManager := range buildClusterManagers {
+		if buildClusterName != "api.ci" {
```
Comment (alvaroaleman) [inline pkg/controller/test-images-distributor/test_images_distributor.go:73]: `== api.ci`
/lgtm cancel

Comment (hongkailiu) [reply pkg/controller/test-images-distributor/test_images_distributor.go]: So close.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**